doc/user/content/sql/system-catalog.md

```diff
@@ -142,6 +142,27 @@ Field            | Type        | Meaning
 `on_expression`  | [`text`]    | If not `NULL`, specifies a SQL expression that is evaluated to compute the value of this index column. The expression may contain references to any of the columns of the relation.
 `nullable`       | [`boolean`] | Can this column of the index evaluate to `NULL`?
 
+### `mz_kafka_consumer_statistics`
+
+The `mz_kafka_consumer_statistics` table contains a row for each Kafka
+consumer in the system.
+
+Field           | Type       | Meaning
+----------------|------------|--------
+`consumer_name` | [`text`]   | The handle name for the consumer.
```

Comment:
I'm not sure what `handle` means is that a standard term?

Reply:
Handle is a term that I grabbed from their documentation.